docs-src/clusters/how-to-set-up-cassandra-visibility-store.md

```diff
@@ -0,0 +1,78 @@
+---
+id: how-to-set-up-cassandra-visibility-store
+title: How to set up Cassandra Visibility store
+sidebar_label: Cassandra
+description: You can set Cassandra as your Visibility store with any other supported Persistence databases.
+tags:
+  - operation-guide
+  - filtered-lists
+  - visibility
+---
+
+You can set Cassandra as your [Visibility store](/concepts/what-is-visibility).
+Verify [supported versions](#supported-databases) before you proceed.
+
+Advanced Visibility is not supported with Cassandra.
+To enable Advanced Visibility features, use any of the supported databases, such as MySQL, PostgreSQL, SQLite, or Elasticsearch, as your Advanced Visibility store.
+We recommend using Elasticsearch for any Temporal Cluster setup that handles more than a few Workflow Executions because it supports the request load on the Visibility store and helps optimize performance.
+
+**Persistence configuration**
+
+Set your Cassandra Visibility store name in the `visibilityStore` parameter in your Persistence configuration, and then define the Visibility store configuration under `datastores`.
+
+The following example shows how to set a Visibility store `cass-visibility` and define the datastore configuration in your Temporal Cluster Configuration YAML.
+
+```yaml
+#...
+persistence:
+  #...
+  visibilityStore: cass-visibility
+  advancedVisibilityStore: es-visibility
```

Comment:
Remove this line.

Reply:
yes! missed it in my last update. is removed now.